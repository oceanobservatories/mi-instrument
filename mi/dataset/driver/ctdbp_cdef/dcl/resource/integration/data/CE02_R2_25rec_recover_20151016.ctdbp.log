2015/10/16 00:00:08.469 [ctdbp:DLOGP3]:Instrument Started [Power On]
2015/10/16 00:00:20.930 # 14.4585,  3.87994,    7.134, 16 Oct 2015 00:00:20
2015/10/16 00:00:30.936 # 14.4384,  3.88064,    7.174, 16 Oct 2015 00:00:30
2015/10/16 00:00:40.925 # 14.4409,  3.88061,    6.981, 16 Oct 2015 00:00:40
2015/10/16 00:00:50.925 # 14.4645,  3.88256,    7.103, 16 Oct 2015 00:00:50
2015/10/16 00:01:00.924 # 14.4671,  3.88218,    7.140, 16 Oct 2015 00:01:00
2015/10/16 00:01:10.924 # 14.4670,  3.88236,    7.296, 16 Oct 2015 00:01:10
2015/10/16 00:01:20.925 # 14.4735,  3.88252,    7.027, 16 Oct 2015 00:01:20
2015/10/16 00:01:30.924 # 14.4764,  3.88283,    7.102, 16 Oct 2015 00:01:30
2015/10/16 00:01:40.925 # 14.4806,  3.88312,    6.951, 16 Oct 2015 00:01:40
2015/10/16 00:01:50.923 # 14.4793,  3.88340,    7.067, 16 Oct 2015 00:01:50
2015/10/16 00:02:00.924 # 14.4797,  3.88336,    7.015, 16 Oct 2015 00:02:00
2015/10/16 00:02:10.924 # 14.4763,  3.88327,    7.200, 16 Oct 2015 00:02:10
2015/10/16 00:02:20.924 # 14.4740,  3.88315,    7.272, 16 Oct 2015 00:02:20
2015/10/16 00:03:25.566 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2015/10/16 00:15:06.489 [ctdbp:DLOGP3]:Instrument Started [Power On]
2015/10/16 00:15:18.930 # 14.3743,  3.87690,    7.398, 16 Oct 2015 00:15:18
2015/10/16 00:15:28.924 # 14.3754,  3.87730,    7.147, 16 Oct 2015 00:15:28
2015/10/16 00:15:38.924 # 14.3838,  3.87759,    7.002, 16 Oct 2015 00:15:38
2015/10/16 00:15:48.924 # 14.3880,  3.87833,    7.110, 16 Oct 2015 00:15:48
2015/10/16 00:15:58.924 # 14.3910,  3.87810,    7.176, 16 Oct 2015 00:15:58
2015/10/16 00:16:08.924 # 14.3974,  3.87851,    6.958, 16 Oct 2015 00:16:08
2015/10/16 00:17:18.924 # 14.3992,  3.87860,    7.116, 16 Oct 2015 00:17:18
2015/10/16 00:17:28.923 # 14.4006,  3.87899,    7.078, 16 Oct 2015 00:17:28
2015/10/16 00:17:38.924 # 14.3954,  3.87874,    7.088, 16 Oct 2015 00:17:38
2015/10/16 00:17:48.924 # 14.3935,  3.87869,    6.832, 16 Oct 2015 00:17:48
2015/10/16 00:17:58.936 # 14.3797,  3.87800,    6.986, 16 Oct 2015 00:17:58
2015/10/16 00:18:08.924 # 14.3784,  3.87789,    7.249, 16 Oct 2015 00:18:08
2015/10/16 00:18:24.237 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
