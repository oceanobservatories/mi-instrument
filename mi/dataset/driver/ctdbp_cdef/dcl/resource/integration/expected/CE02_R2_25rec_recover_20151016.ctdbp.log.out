
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942420.9300003052 |          2 |         3653942420 | 3653942420.9300003052 |      port_timestamp |           ok |      3.87994 |             null |                     null |    7.134 | 14.4585
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942430.9359998703 |          2 |         3653942430 | 3653942430.9359998703 |      port_timestamp |           ok |      3.88064 |             null |                     null |    7.174 | 14.4384
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942440.9250001907 |          2 |         3653942440 | 3653942440.9250001907 |      port_timestamp |           ok |      3.88061 |             null |                     null |    6.981 | 14.4409
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942450.9250001907 |          2 |         3653942450 | 3653942450.9250001907 |      port_timestamp |           ok |      3.88256 |             null |                     null |    7.103 | 14.4645
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942460.9239997864 |          2 |         3653942460 | 3653942460.9239997864 |      port_timestamp |           ok |      3.88218 |             null |                     null |     7.14 | 14.4671
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942470.9239997864 |          2 |         3653942470 | 3653942470.9239997864 |      port_timestamp |           ok |      3.88236 |             null |                     null |    7.296 |  14.467
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942480.9250001907 |          2 |         3653942480 | 3653942480.9250001907 |      port_timestamp |           ok |      3.88252 |             null |                     null |    7.027 | 14.4735
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942490.9239997864 |          2 |         3653942490 | 3653942490.9239997864 |      port_timestamp |           ok |      3.88283 |             null |                     null |    7.102 | 14.4764
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942500.9250001907 |          2 |         3653942500 | 3653942500.9250001907 |      port_timestamp |           ok |      3.88312 |             null |                     null |    6.951 | 14.4806
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942510.9230003357 |          2 |         3653942510 | 3653942510.9230003357 |      port_timestamp |           ok |       3.8834 |             null |                     null |    7.067 | 14.4793
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942520.9239997864 |          2 |         3653942520 | 3653942520.9239997864 |      port_timestamp |           ok |      3.88336 |             null |                     null |    7.015 | 14.4797
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942530.9239997864 |          2 |         3653942530 | 3653942530.9239997864 |      port_timestamp |           ok |      3.88327 |             null |                     null |      7.2 | 14.4763
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653942540.9239997864 |          2 |         3653942540 | 3653942540.9239997864 |      port_timestamp |           ok |      3.88315 |             null |                     null |    7.272 |  14.474
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943318.9300003052 |          2 |         3653943318 | 3653943318.9300003052 |      port_timestamp |           ok |       3.8769 |             null |                     null |    7.398 | 14.3743
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943328.9239997864 |          2 |         3653943328 | 3653943328.9239997864 |      port_timestamp |           ok |       3.8773 |             null |                     null |    7.147 | 14.3754
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943338.9239997864 |          2 |         3653943338 | 3653943338.9239997864 |      port_timestamp |           ok |      3.87759 |             null |                     null |    7.002 | 14.3838
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943348.9239997864 |          2 |         3653943348 | 3653943348.9239997864 |      port_timestamp |           ok |      3.87833 |             null |                     null |     7.11 |  14.388
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943358.9239997864 |          2 |         3653943358 | 3653943358.9239997864 |      port_timestamp |           ok |       3.8781 |             null |                     null |    7.176 |  14.391
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943368.9239997864 |          2 |         3653943368 | 3653943368.9239997864 |      port_timestamp |           ok |      3.87851 |             null |                     null |    6.958 | 14.3974
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943438.9239997864 |          2 |         3653943438 | 3653943438.9239997864 |      port_timestamp |           ok |       3.8786 |             null |                     null |    7.116 | 14.3992
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943448.9230003357 |          2 |         3653943448 | 3653943448.9230003357 |      port_timestamp |           ok |      3.87899 |             null |                     null |    7.078 | 14.4006
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943458.9239997864 |          2 |         3653943458 | 3653943458.9239997864 |      port_timestamp |           ok |      3.87874 |             null |                     null |    7.088 | 14.3954
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943468.9239997864 |          2 |         3653943468 | 3653943468.9239997864 |      port_timestamp |           ok |      3.87869 |             null |                     null |    6.832 | 14.3935
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943478.9359998703 |          2 |         3653943478 | 3653943478.9359998703 |      port_timestamp |           ok |        3.878 |             null |                     null |    6.986 | 14.3797
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653942400 | recovered | 3653943488.9239997864 |          2 |         3653943488 | 3653943488.9239997864 |      port_timestamp |           ok |      3.87789 |             null |                     null |    7.249 | 14.3784

(25 rows)
